src/sgame/sg_admin.cpp

```diff
@@ -5925,8 +5925,8 @@ static inline void G_admin_buffer_print_raw( gentity_t *ent, Str::StringRef m, b
 		}
 	}
 
-	// Ensure we don't overflow client buffers. Allow for surrounding quotes and each needed escaping backslash
-	if ( g_bfb.size() + m.size() + bfbNaughtyCharacters + additionalNaughtyCharacters + 2 >= MAX_MESSAGE_SIZE )
+	// Ensure we don't overflow client buffers. Allow for 'print' command, surrounding quotes and each needed escaping backslash
+	if ( strlen("print ") + g_bfb.size() + m.size() + bfbNaughtyCharacters + additionalNaughtyCharacters + 2 >= MAX_MESSAGE_SIZE)
```

Comment:
Will this be done at compile time or is this premature optimization?

Reply:
I believe so (and yes :stuck_out_tongue: )